pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -17,6 +17,13 @@
 , openssl
 , expat
 , libxcrypt-legacy
+, coreutils
+, gnugrep
+, which
+, git
+, libsecret
+, e2fsprogs
+, libnotify
```

Comment:
Is this still needed, now that the other wrapper has been removed?

Reply:
Great catch! I have removed them now, and tested just to be sure!